code/modules/antagonists/changeling/powers/shriek.dm

```diff
@@ -10,6 +10,9 @@
 //A flashy ability, good for crowd control and sowing chaos.
 /datum/action/changeling/resonant_shriek/sting_action(mob/user)
 	..()
+	if(user.movement_type & VENTCRAWLING)
+		to_chat(user, span_notice("We must exit the pipes before we can shriek, otherwise it would just reverberate down the pipes ineffectually!"))
```

Comment:
This is nuclear level nitpicking but a failure message like this should be under a warning span

Reply:
it should actually be a balloon alert

Reply:
that works too